### ISSUE TITLE
Feature Request: Smart domain sort in Dashboard (sort by domain level)

### ISSUE THREAD

**Issue by sshipway** [PowerDNS-Admin/PowerDNS-Admin#729]
Currently, when sorting by name, it treats the field as one.  Howver, if entries have subdomains, this is not the ideal way to sort, and it would be better to sort within each subdomain.

*Example:*

Current ordering (fully alphabetic)
a
a.b
b.z
d
x.z
z.b

Preferred ordering (within subdomain)
a
a.b
z.b
d
x.z
b.z

This sorts by name components, right to left, so: a, b, d, z.  Then it sorts within the b and z domains.
This will make it easier to view a large zone file, as subdomain records will be groups appropriately.
In addition, reverse zones should sort *numerically* rather than alphabetically, thus:

5
6
11
12
100
250

Ideally, this would be either a global option, or a secondary sort ordering (so click on the heading once to reverse direction, then again to change sort mode, etc)

This is most useful when viewing reverse zones at a class B level as they can get hard to follow otherwise.

**Comment by jbe-dw** [comment]
I can see that you have a lot of ideas to improve PDA.

**Comment by jbe-dw** [comment]
Hello,

This request is stale for too long. Regarding our policy to cleanup this project, we are closing it. As this is a Feature Request, it will be listed in #1178

Regards
PowerDNS Admin Team
See #1153 for the policy
